### PR TITLE
pprof requires binary as well as pprof output when using files.

### PR DESCRIPTION
**Pull request by prashantv**
from pprof usage:
  usage: pprof [options] [binary] <profile source> ...

require a binaryname arg when using binaryfile.
verified this with local -cpuprofile output from tests.

r @sandlerben

### CODE REVIEW

**Comment by sectioneight** [comment]
Nit: let's get our commit messages up to open source standards. Capitalize, short subject, imperative verb, no trailing period.

http://chris.beams.io/posts/git-commit/

File: main.go [hunk 1]
```diff
@@ -112,6 +112,11 @@ func (t *torcher) createAndRunApp() {
 			Usage: "file path of raw binary profile; alternative to having go-torch query pprof endpoint " +
 				"(binary profile is anything accepted by https://golang.org/cmd/pprof)",
 		},
+		cli.StringFlag{
```
Comment (sandlerben) [inline main.go]: I would make this optional. When we give raw pb.gz files, the binary name is not needed.

File: main.go [hunk 2]
```diff
@@ -152,14 +158,17 @@ func (com *defaultCommander) goTorchCommand(c *cli.Context) {
...
+	var pprofArgs []string
 	if binaryInput != "" {
-		profileSource = binaryInput
+		if binaryName == "" {
+			log.Fatalf("binaryname is required when specifying binaryinput")
+		}
+		pprofArgs = []string{binaryName, binaryInput}
 	} else {
-		profileSource = url
+		pprofArgs = []string{"-seconds", fmt.Sprint(time), url}
```
Comment (sandlerben) [inline main.go]: Did you verify that the time argument still works like this?

File: main.go [hunk 3]
```diff
@@ -152,14 +158,17 @@ func (com *defaultCommander) goTorchCommand(c *cli.Context) {
...
 	}
 
-	out, err := com.pprofer.runPprofCommand(time, profileSource)
+	out, err := com.pprofer.runPprofCommand(pprofArgs...)
```
Comment (sandlerben) [inline main.go]: I like this better.

**Comment by sandlerben** [comment]
lgtm once you make binaryfile optional

File: main.go (hunk 2 above)
Comment (prashantv) [inline main.go]: Yep, verified, works fine.

**Comment by prashantv** [comment]
Updated commit messages + made binaryname optional so the .pb.gz files don't need it.

**Pull request merged**